### ISSUE TITLE
How to excecute this

### ISSUE THREAD

**Issue by Naeto11** [2Math0/apple_lock_down_safari_alert#1]
Could you please provide a step-by-step process as to how you execute this and also the apps or website that will be used for this.

**Comment by 2Math1** [comment]
- Follow the read me file 
- add the alert handler in the web folder
- add the line of meta in the index.html body

**Comment by 2Math0** [comment]
it's pretty simple just follow the readme 
take the tag and put it in the index.html you will find it inside the web folder inside your project directory and wallah

**Comment by Naeto11** [comment]
I would please like to know if this can help with the bypassing of respondus Lockdown browser on the mac

**Comment by 2Math0** [comment]
> I would please like to know if this can help with the bypassing of respondus Lockdown browser on the mac

it won't directly bypass the lockdown,
I have done my search and apparently, lockdown refuses Flutter Web completely

My Take was: the alternative solution is to give a dialog box for user telling them that they need to turn off lockdown to be able to continue browsing this website

and for sure I had to make the dialog box in js and CSS purely to be able to be shown for user without getting affected by lockdown policy

**Comment by Naeto11** [comment]
I would like to know how this part can be done and if we could break them down in a step-by-step process. I not an IT major so I find it hard to follow 😓: My Take was that the alternative solution is to give a dialog box for users telling them that they need to turn off lockdown to able to continue browsing this website

and for sure I had to make the dialog box in js and CSS purely to be able to be shown for user without getting affected by lockdown policy

**Comment by 2Math0** [comment]
yes, just include the library of this package inside index.html same exactly as in the read me you will see the result instantly after run

**Comment by Naeto11** [comment]
I would like to know what index.html file you mean specifically. In the folder [apple_lock_down_safari_alert-main], there is no index.html file
